tarantool/response.py

```diff
@@ -248,3 +251,34 @@ def __str__(self):
         return ''.join(output)
 
     __repr__ = __str__
+
+
+class ResponseExecute(Response):
+    @property
+    def lastrowid(self):
+        if self.body is None:
+            raise InterfaceError("Trying to access data, when there's no data")
+        info = self.body.get(IPROTO_SQL_INFO)
+
+        if info is None:
+            return None
+
+        lastrowids = info.get(IPROTO_SQL_INFO_AUTOINCREMENT_IDS)
+
+        return lastrowids[-1] if lastrowids else None
+
+    @property
+    def rowcount(self):
+        if self._body is None:
+            raise InterfaceError("Trying to access data, when there's no data")
+
+        info = self._body.get(IPROTO_SQL_INFO)
+
+        if info is None:
+            return -1
```

Comment:
Can not agree or I didn't get something because I could't find `IPROTO_SQL_INFO` in DQL response. Thus, I think the current implementation fits for all cases.

In case of dql the response body looks like:
```
 {IPROTO_METADATA: [field_meta_dicts], IPROTO_DATA: [rows]}
```
dml request:
```
{IPROTO_SQL_INFO: {IPROTO_SQL_INFO_ROW_COUNT: count, IPROTO_SQL_INFO_AUTOINCREMENT_IDS: [id]}}
```